x/proof/keeper/msg_server_update_param_test.go

```diff
@@ -40,7 +40,7 @@ func TestMsgUpdateParam_UpdateMinRelayDifficultyBitsOnly(t *testing.T) {
 	require.Equal(t, expectedRelayDifficultyTargetHash, res.Params.RelayDifficultyTargetHash)
 
 	// Ensure the other parameters are unchanged
-	testkeeper.AssertDefaultParamsEqualExceptFields(t, &defaultParams, res.Params, "RelayDifficultyTargetHash")
+	testkeeper.AssertDefaultParamsEqualExceptFields(t, &defaultParams, res.Params, prooftypes.ParamRelayDifficultyTargetHash)
```

Comment:
```suggestion
	testkeeper.AssertDefaultParamsEqualExceptFields(t, &defaultParams, res.Params, string(prooftypes.KeyRelayDifficultyTargetHash))
```

Same for the rest...